OOPP/app/src/main/java/com/example/sauronsarmy/oopp/MainActivity.java

```diff
@@ -7,13 +7,22 @@
 import android.view.View;
 import android.widget.ImageButton;
 
-public class MainActivity extends AppCompatActivity {
+public class MainActivity extends AppCompatActivity implements MainMVPInterface.ViewOps {
+
+    private MainMVPInterface.PresenterOps mainPresenter;
+
+    @Override
+    public void showAlert(String msg){}//To be implemented
```

Comment:
Jag lade egentligen bara in dem som typ "standard" view metoder, kan hända att de tas bort!